config/locales/js-en.yml

```diff
@@ -82,6 +83,7 @@ en:
     label_collapse: "Collapse"
     label_collapsed: "collapsed"
     label_collapse_all: "Collapse all"
+    label_committed_at: "committed revision %{revision}"
```

Comment:
The key and value do not match.
I would expect a value of *committed revision %{revision}* ***at***